app/webpacker/components/RegistrationsV2/Register/CompetingStep.jsx

```diff
@@ -31,7 +32,7 @@ const maxCommentLength = 240;
 const potentialWarnings = (competitionInfo) => {
   const warnings = [];
   // Organizer Pre Registration
-  if (!competitionInfo['registration_currently_open?']) {
+  if (DateTime.fromISO(competitionInfo.registration_open) > DateTime.now()) {
```

Comment:
Too much parsing logic inside an `if` clause. I feel like we already have a method for that, let me check...

Reply:
Yes, we have `hasPassed` and `hasNotPassed` in `lib/utils/dates.js`

Reply:
done